views/json/resources/default.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+echo json_encode($vars);
```

Comment:
Here we would implement some sort of serializer. IMO, we should make `ElggEntity` implement `Serializable`